subt/drone.py

```diff
@@ -102,6 +113,9 @@ def on_pose3d(self, data):
     def on_air_pressure(self, data):
         self.altitude = altitude_from_pressure(data)
 
+    def on_desired_altitude(self, data):
+        pass  # self.desired_altitude already updated by Node
+
```

Comment:
Do we really need this here?

Reply:
no we do not, but ... as you mentioned in some other PR, it is not obvious that `self.desired_altitude` is automatically updated. So for me it is just a note - I can remove it if it is rather misleading?

Reply:
no big deal, keep it